_assets/style.scss

```diff
@@ -192,13 +193,25 @@ ul, li {
 }
 
 .alert { font-size: 13px; white-space: nowrap; text-overflow: ellipsis; overflow: hidden; display: none; position: fixed; margin: auto; left: 50%; width: 200px; margin-left: -100px; top: 10px; background: rgba(10,50,90,0.7); color: #fff; font-weight: bold; padding: 7px; text-align: center; border-radius: 5px; }
+
+.native-emoji,
 .emoji {
   @include inline-block;
+  cursor: pointer;
+}
+
+.native-emoji {
+  font-size: 19px;
+  padding: 10px 2px;
+  font-family: AppleColorEmoji;
```

Comment:
Had to add this because some are not being rendered as colored emoji. For example: `☺️` 
but seems that obviously it won't work on windows and others, will probably need to test it out and find a solution for each OS :/